bin/fillTestData/Messages.js

```diff
@@ -4,8 +4,27 @@ var _ = require('lodash'),
     path = require('path'),
     mongooseService = require(path.resolve('./config/lib/mongoose')),
     chalk = require('chalk'),
+    argv = require('yargs')
```

Comment:
For  clarity, let's separate requiring the module and configuring it;
```js
argv = require('yargs');
```

and further down in the app:
```js
argv
  .usage()
  // ...etc
```